src/components/Dialog.scss

```diff
@@ -1,6 +1,11 @@
 @import "../css/_variables";
 
 .excalidraw {
+  .Dialog {
+    user-select: text;
+    cursor: text;
```

Comment:
This affects the whole dialog.. can we remove the cursor?

Reply:
but I will show cross hair cursor when user hovers over the modal

Reply:
I think I could add a new class for keyboard dialog and add that cursor there,

Reply:
That might be better alternative. Any thoughts ?